test/MiniMeToken.t.sol

```diff
@@ -46,21 +64,110 @@ contract MiniMeTokenTest is Test {
     }
 }
 
+contract AcceptingController is TokenController {
+    receive() external payable { }
```

Comment:
Might be worth looking into so the linter doesn't complain about empty blocks in `receive()` (or `fallback()`)